utils/jsonhex/types.go

```diff
@@ -23,7 +23,7 @@ func (h *Bytes) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
-	if s == "null"{
+	if s == `` {
```

Comment:
I think the empty string should be treated like a parsing error. After all, there is no `Bytes` instance that could be marshaled into an empty string.

Reply:
the problem is that the JSON encoding of "null" is unmarshaled to the empty string. This is why I had to make this change

Reply:
using `json.RawMessage` we can see the null instead of losing it. For unmarshaling of other types, using `json.RawMessage` and then `string.Trim(\`\"\`)` would allow us to accept data values with and without `""`.